arch/risc-v/src/common/riscv_fork.c

```diff
@@ -102,6 +103,7 @@
 
 pid_t riscv_fork(const struct fork_s *context)
 {
+  uintreg_t *parent_regs = (uintreg_t *)riscv_percpu_get_regs();
```

Comment:
Please remove the fork() related fixes from this patch, as they are not related to the topic of this PR or the patch.

Reply:
ok